db/migrate/20170808225329_create_addresses.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAddresses < ActiveRecord::Migration[5.1]
+  def change
+    create_table :addresses do |t|
+      t.timestamps null: false
+      t.string :street_address
+      t.string :city
+      t.string :county
+      t.string :state
+      t.string :zip
+      t.boolean :mailing
+      t.belongs_to :snap_application, index: true
+    end
```

Comment:
Is there anything here we should enforce with some `null: false` constraints? Looking at `app/steps/mailing_address.rb` it seems as if some of these could use some validation on the db side for data integrity purposes.